### DISCUSSION TITLE
Error creating next app

### DISCUSSION THREAD

**Discussion by PranganTooteja** [vercel/next.js#15014] [Help]
Windows 10 Node 12.18.2

npm ERR! Unexpected end of JSON input while parsing near '...0-beta.40":{"name":"@'

silly fetchPackageMetaData error for @babel/core@7.7.7 Unexpected end of JSON input while parsing near '...":"7.0.0-beta.41","ba'

Please help!

**Comment by jamesmosier** [comment]
Could you share some more information? What are you trying to do? What does your `package.json` look like?
